Scripts/gen_Primary.sh

```diff
@@ -26,16 +25,14 @@ ENDMOM=110 # optional (for flat only)
 OWNER=mu2e
 RUN=1202
 CAT="Cat"
-TAG=""
 
 # Function: Print a help message.
 usage() {
   echo "Usage: $0
   [ --primary primary physics name ]
-  [ --campaign primary camapign name ]
-  [ --scampaign stops camapign name ]
```

Comment:
Hi Yuri, we need to retain the possibility of different campaigns for in and output